### DISCUSSION TITLE
Which columns should be considered to be filtered for precursor quantification?

### DISCUSSION THREAD

**Discussion by MaldackerM** [vdemichev/DiaNN#953] [Q&A]
Hello,

i am currently running some analysis that require to filter for high confidence peptides that should be quantified. During this - while scrolling through the main output report from DIANN - I wondered which values are to consider for filtering. For example, what are the columns of Quantity.Quality, MS1.Profile.Corr, Mass.Evidence, Averagine and Decoy.Evidence telling me and what are considerable cutoff values that allow me stringent filtering to have high confidence quantification and identification results.

I know that has been partly discussed before, but I also wondered if there is any kind of interplay for these values.

Thanks in advance for any help!

**Comment by vdemichev** [comment]
PEP filter is enough to ensure general identification confidence. A combination of Ms1.Profile.Corr and Mass.Evidence can be used to boost peptidoform mass assignment confidence. If you have isolated runs where you need to verify confident identification, you can export chromatograms with --vis and also check them manually.
